### DISCUSSION TITLE
[Sample] Left Navbar with content

### DISCUSSION THREAD

**Discussion by dcdh** [mantinedev/mantine#6267] [Q&A]
Hello all,

I am trying to use the double navbar available here https://ui.mantine.dev/component/double-navbar/
with a content.

Unfortunately I am unable to make it on the right of the left navbar. The content is at the bottom.
After the `<nav/>` I am trying to add `<main>Content</main>`

Can you explain me how to do it. Thanks

**Comment by dongliangCode** [comment]
Excuse me, have you found a solution?

**Comment by dongliangCode** [comment]
We also need this feature
